Article first draft (#135)
hassanobeid1994/tr_b_causal_2020: article/sections/_1_intro_generic.tex

\section{Introduction}
\label{sec:intro}

> The following chapter is concerned with the existing disconnect between the 

Author: ```suggestion
This chapter is concerned with the existing disconnect between the 
```

Author: Could you reformat the sentence so it uses active rather than passive voice? This feedback applies to all of your sections, but especially for the first sentence of the chapter!

> fields of travel demand modeling and causal inference. 

Author: It's the handbook of "choice" modelling so the interest is larger than just transportation (though it's of course important, given the editors and history of choice modelling).
```suggestion
fields of choice modeling and causal inference. 
```

> More specifically, the chapter is motivated by the current lack of use of methods and findings from 

Author: ```suggestion
Specifically, the chapter is motivated by the current lack of use of methods and findings from 
```

Author: Maybe this is a better suggestion than above. I changed "current lack of use" to "routine absence" because its a typical not total lack of use.
```suggestion
Specifically, the chapter is motivated by the routine absence of methods and findings from 
```

the causal inference literature in travel demand modeling. 

> More often than not, the development of behavioral models in transportation, 

Author: ```suggestion
In transportation, the development of behavioral models, 
```

Author: This sentence has many parenthetical statements, making it hard to follow (in my opinion). Could you split the sentence to make it easier?

Perhaps something along the lines of:

"In transportation, we often build models for program / policy evaluation.
Specifically, we use behavioral models to evaluate the impact of external interventions on travel outcomes.
These evaluations are explicitly causal: we are interested in [...]."

specifically transportation demand models, is driven by the need to evaluate 

> the impact external interventions, in the form of alternative policies, on a 

Author: ```suggestion
the impact of external interventions, in the form of alternative policies, on a 
```

certain outcome of interest. 
These questions that transportation demand models 
are built to answer are causal by definition: we are interested in how the 

> system reacts to \textit{external} interventions. 

Author: Should external be the point of emphasis? For contrast, what is an internal intervention? I think it would be fine (best?) to remove the word "external" and instead italicize "interventions". I think it's the very notion of intervening that sets the analysis apart from passive, observational studies.

Yet, when those models are 

> developed, there is very little consideration given to causality, and when 

Author: ```suggestion
developed, causality is often not considered. Furthermore, when 
```

> causal concepts are accounted for, the process is done implicitly without a 

Author: ```suggestion
causal concepts are accounted for, the process is often implicit and without a 
```

formal framework. 

While the field of transportation demand modeling could benefit greatly from 

> incorporating causal inference techniques, there are barriers that have made 
> this integration slower than what one would hope. 

Author: ```suggestion
incorporating causal inference techniques, there are barriers that have slowed this integration. 
```

These barriers stem from the difference between the types of problems transportation demand modelers deal with and those that are typically studied in the causal inference literature. 

> Perhaps the main fundemental difference is that demand modelers 

Author: ```suggestion
Perhaps the fundamental difference is that demand modelers 
```

> are typically trying to forecast the impacts of policies that haven't been 
> implemented or seen before, which requires additional work and a change to the
> typical causal modeling workflow in order to translate a given policy 
> (treatment) into a set of characteristics and variables that exist in the
> data and system at hand (please refer to \citet{brathwaite_2018_causal} for a 
> more thorough discussion of those barriers). 

Author: ```suggestion
are typically trying to forecast the impacts of policies that haven't been implemented or seen before.
Forecasting the effects of unseen interventions requires additional work and a change to the typical causal modeling workflow.
In particular, we must translate a given policy (treatment) into a set of characteristics and variables that exist in the data and system at hand.
(For a more thorough discussion of this and other barriers, please refer to \citet{brathwaite_2018_causal}.) 
```

> While those barriers make the problem of demand modelers harder, there is still a lot to gain from 
> incorporating causal inference techniques where appropriate, and to contribute
> in turn where the literature lacks.  

Author: ```suggestion
While these barriers complicate the efforts of demand modelers, there is still a lot to gain from
incorporating causal inference techniques where appropriate and from contributing
to the causal inference literature where it's lacking.
```

> The relevance of this topic now is motivated by the significant boost in the causal inference literature recently, both in the potential outcomes framework and the causal graphical approach.

Author: Can you change this sentence from passive to active voice? I think this sentence is trying to say "causal inference is an increasingly relevant topic because research on the topic is accelerating."

Feel free to put whatever you mean, but state it actively. Also, I think the field of causal inference is increasingly relevant not just because there is more and more research in it, but also because there is more and more research that is useful to choice modelers (e.g. trying to handle unobserved confounding).

The goal of this chapter is to formalize a framework for approaching 
transportation demand modeling problems from a causal perspective. 

> We will draw heavily on the use of directed acyclic graphs (DAGs) formalized by \citet{pearl_causality_2000} as a means of representing the modeler's knowledge and assumptions about a given problem. 

Author: To the extent possible, let's refer to the chapter in its whole in the present tense. I.e., "We draw heavily [...]" as opposed to "We will draw heavily".

If necessary, we'll use the future tense. For instance, in line 90 of section four where I say "In the next section, we'll [...]." There's not really a way out of the future tense there (but let me know if you see it!).

The main reason is to enforce consistency across us as much as possible, and to use less words. Future tense leads to lots of "will" words throughout the document.
```suggestion
We draw heavily on the use of directed acyclic graphs (DAGs) formalized by \citet{pearl_causality_2000} as a means of representing the modeler's knowledge and assumptions about a given problem. 
```

The chapter will provide an overview of DAGs, the 
testable implications that come with one's causal representation, the main 
tests that one could do to falsify/justify a given causal graph, and then how 
to use a causal graph to estimate the causal relationships of interest. 
We will demonstrate the use of this framework through simulations, where we 
clearly show the benefits and implications of this approach as opposed to 
traditional approaches. 
The last part of this chapter deals with the more complicated issue of latent 
confounding, where one variable confounds two or more variables in the causal 
graph. This type of confounding creates variations in the outcome variable that are not caused by 
the confounded variables but are correlated by it, which biases the estimated 
causal effects of those variables if nothing is done to account for the 

> confounding. 

Author: Can you reference the sections in which each of these things happen? E.g. "testable implications that come with one's causal representation (Section 5)". I think this paragraph should serve as the typical "In Section __ we ___."
